src/widget/OktaSignIn.js

```diff
@@ -117,9 +116,21 @@ var OktaSignIn = (function () {
     require.config(requireConfig);
     OktaAuth = require('vendor/OktaAuth');
     Util = require('util/Util');
+    LoginRouter = require('LoginRouter');
+
 
     authClient = new OktaAuth({ uri: options.baseUrl, transformErrorXHR: Util.transformErrorXHR });
-    _.extend(this, getProperties(authClient, options));
+    _.extend(this, LoginRouter.prototype.Events, getProperties(authClient, options, LoginRouter));
+
+    // Triggers the event up the chain so it is available to the consumers of the widget.
+    this.listenTo(LoginRouter.prototype, 'all', function() {
+      this.trigger.apply(this, arguments);
+    });
+
+    this.on('pageRendered', function(classname) {
```

Comment:
Remove this. For testing, it's probably better to just test from the consumer side anyway (i.e. if you're working in okta-signin-widget, you can modify https://github.com/okta/okta-signin-widget/blob/master/buildtools/templates/index.tpl to make sure the actual widget is publishing those events.